src/main/zapHomeFiles/hud/tools/toggleScript.js

```diff
@@ -0,0 +1,188 @@
+/*
+ * Toggle Script tool
+ *
+ * When selected, allows a user to choose a Script to toggle.
+ * Togglable scripts MUST have an enabled property in order to be selected.
+ * When left-clicked after a script is selected, will toggle that script to its opposite state.
+ */
+
+const ToggleScript = (function () {
+	// Constants
+	const NAME = 'toggleScript';
+	const LABEL = I18n.t('togglescript_tool');
+	const ICON = {};
+	ICON.DEFAULT = 'script-add.png';
+	ICON.ON = 'script-enabled.png';
+	ICON.OFF = 'script-disabled.png';
+	const DATA = {};
+	DATA.ON = I18n.t('common_on');
+	DATA.OFF = I18n.t('common.off');
```

Comment:
Should be common_off ? It doesnt display properly for me :)

Reply:
Whoops, thanks for the catch